### DISCUSSION TITLE
Hey, guys! Can anyone help me with a question? I want to know the difference between 'magma-galaxy' and 'magma.

### DISCUSSION THREAD

**Discussion by Breno-Pimentel** [magma/magma#15358] [General]
Hey, guys! Can anyone help me with a question? I want to know the difference between 'magma-galaxy' and 'magma.

**Comment by lucasgonze** [comment]
Hi. Thanks for moving this to here from Slack.

magma-galaxy is an independent project created by Shubham Tatvmasi (@ShubhamTatvamasi), who was a big contributor to Magma at that time. We had hoped to absorb it into Magma proper but that hasn't happened. 

magma-galaxy is fundamentally a wrapper script. Per [the home page on Ansible Galaxy](https://galaxy.ansible.com/ui/repo/published/shubhamtatvamasi/magma/) it is a "Magma collection for deploying Orchestrator"
